docs/config.md

```diff
@@ -13,12 +13,136 @@ SPDX-License-Identifier: Apache-2.0
             
 - **gnbs:** 
     List of gNB's to be simulated. Each item in the list holds configuration specific to a gNB.
+    Support of multiple gNBs: Two gNBs are configured by default. So, User can create profiles by using these gNBs.
+    Configuration of two gNBs can be found here
+
+        gnb:
+          ips:
+          - '"192.168.251.5/24"' #gnb1 IP
+          - '"192.168.251.6/32"' #gnb2 IP
+        configuration:
+          runConfigProfilesAtStart: true
+          singleInterface: #this will be added through configmap script
+          execInParallel: false #run all profiles in parallel
```

Comment:
```suggestion
          singleInterface: # this will be added through configmap script
          execInParallel: false # run all profiles in parallel
```